app/models/vote.rb

```diff
@@ -0,0 +1,21 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  validates_presence_of :user
+  validates_presence_of :review
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.